src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -381,33 +388,31 @@ private AIState requestMaterials()
     {
         //todo as soon as material handling has been implemented this should be set to work!
         //We need to deal with materials
-        if(!Configurations.builderInfiniteResources)
+        if (!Configurations.builderInfiniteResources)
         {
             //TODO thread this
-            while(job.getSchematic().findNextBlock())
+            while (job.getSchematic().findNextBlock())
             {
-                if(job.getSchematic().doesSchematicBlockEqualWorldBlock())
+                if (job.getSchematic().doesSchematicBlockEqualWorldBlock())
                 {
                     continue;
                 }
 
-                Block block = job.getSchematic().getBlock();
+                Block     block     = job.getSchematic().getBlock();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'block' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)